repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/ChallengeQuestionTabEntity.java

```diff
@@ -0,0 +1,115 @@
+package uk.gov.hmcts.ccd.definition.store.repository.entity;
+
+import java.io.Serializable;
+
+import static javax.persistence.FetchType.LAZY;
+
+import javax.persistence.*;
+
+@Table(name = "new_challenge_question")
+@Entity
+public class ChallengeQuestionTabEntity implements Serializable {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne(fetch = LAZY)
+    @JoinColumn(name = "case_type_id", nullable = false)
+    private CaseTypeEntity caseType;
+
+    @Column(name = "display_order")
+    private Integer order;
+
+    @Column(name = "question_text", nullable = false)
+    private String questionText;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "answer_field_type", nullable = false)
+    private FieldTypeEntity answerFieldType;
+
+    @Column(name = "display_context_parameter")
+    private String displayContextParameter;
+
+    @Column(name = "challenge_question_id", nullable = false)
+    private String challengeQuestionId;
+
+    @Column(name = "answer_field", nullable = false)
+    private String answerField;
+
+    @Column(name = "question_id")
+    private String  questionId;
```

Comment:
Is `questionId` needed on top of `challengeQuestionId`?

Reply:
Yes Dan one is for the group of questions other of one question.

Reply:
Thanks, didn't understand the difference but do now after a 3A!